src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -20,56 +24,49 @@
-    ObiSkills.add("HTML wrangler");
-    MentorObject Obi =
-        new PersonBuilder()
-            .name("Obi")
-            .gitHubID("Obinnabii")
-            .description("Obi is awesome.")
-            .interestTags(ObiSkills)
-            .projectIDs(projects)
-            .buildMentor();
-    mentors.add(Obi);
-
-    ArrayList<String> SamiSkills = new ArrayList<String>();
-    SamiSkills.add("Stone carver");
-    SamiSkills.add("Bootstrap convert");
-    MentorObject Sami =
-        new PersonBuilder()
-            .name("Sami")
-            .gitHubID("Sami-2000")
-            .description("Sami is fun.")
-            .interestTags(SamiSkills)
-            .projectIDs(projects)
-            .buildMentor();
-    mentors.add(Sami);
-
-    ArrayList<String> RichiSkills = new ArrayList<String>();
-    RichiSkills.add("Minecraft boss");
-    RichiSkills.add("React wizard");
-    MentorObject Richi =
-        new PersonBuilder()
-            .name("Richi")
-            .gitHubID("Richie78321")
-            .description("Richi is cool.")
-            .interestTags(RichiSkills)
-            .projectIDs(projects)
-            .buildMentor();
-    mentors.add(Richi);
-
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    List<UserEntity> mentorEntities = ofy().load().type(UserEntity.class).list();
+    ArrayList<UserData> mentors = new ArrayList<UserData>();
+    for (UserEntity entity : mentorEntities) {
+      if (entity.isMentor()) {
+        mentors.add(new UserData(entity));
+      }
+    }
     String jsonMentors = new Gson().toJson(mentors);
     response.setContentType("application/json;");
     response.getWriter().println(jsonMentors);
   }
 
-  // This function sends an error if invalid input is found.
+  // Add a mock Sami object to the datastore.
+  public void addMockMentor(HttpServletResponse response) throws ServletException, IOException {
+    GHRepository testRepo;
+    GitHub gitHub = GitHubGetter.getGitHub();
+    try {
+      testRepo = gitHub.getRepository("googleinterns/step22-2020");
+    } catch (Exception e) {
+      error(response, "Failed to access test repo.", 400, "Test repo not found");
+      return;
+    }
+    Long testRepoID = testRepo.getId();
+    ProjectEntity testProject = ProjectEntity.fromRepositoryIdOrNew(testRepoID.toString());
+    String id = "mock_id";
+    if (!testProject.mentorIds.contains(id)) {
+      testProject.mentorIds.add(id);
+    }
+    ofy().save().entity(testProject);
+
+    ArrayList<String> interests = new ArrayList<String>();
+    interests.add("skateboarding");
+    ArrayList<String> mentees = new ArrayList<String>();
+    ArrayList<String> projects = new ArrayList<String>();
+    projects.add(testRepoID.toString());
+    UserEntity mockMentor =
+        new UserEntity(id, "Sami-2000", interests, "samialves@google.com", projects, mentees);
+    ofy().save().entity(mockMentor);
+  }
+
```

Comment:
If I remember correctly, in the demo someone said that we can now add people to the datastore just by them signing up. If so can we delete this function?